docs/concepts/Deploy-Command.md

```diff
@@ -54,6 +54,9 @@ To provision the backend services which are used to execute customized logics fo
 * If failure happens in one region, CLI won't rollback the changes. Please try with a further deploy which contains the fix to the failure.
 * This step is skippable when you remove the entire `skillInfrastructure` field from ask-resources config file.
 
+### Deploy only skill package 
+ASK CLI v1 had option '--target' to deploy only skill package and not the infrastructure. Similar behavior can be achieved in ASK CLI v2 by removing "skillInfrastructure" property from ask-resources.json. 
+
```

Comment:
It's really nice to mention this to users, it's a great value~ should this be put to FAQ? With this topic around how to deploy skill package as the only target. Reason is this talks about v1 staff so it makes me think FAQ would be a better place